virttest/utils_test/libvirt.py

```diff
@@ -1168,6 +1168,9 @@ def pre_pool(self, pool_name, pool_type, pool_target, emulated_image,
         xml_str = virsh.pool_dumpxml(pool_name)
         logging.debug("New prepared pool XML: %s", xml_str)
 
+        logging.info("refresh pool")
```

Comment:
Can you capitalize this and write it as a whole sentence like
```python
logging.info("Refreshing pool")
```

Reply:
Fixed, thanks